portfolio.html

```diff
@@ -139,6 +139,585 @@ <h2 class="description">I'm a software developer</h2>
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="about-section">
+    <div class="about-text">
+      <h1 class="primary-text">About me</h1>
```

Comment:
Please change this tag to any other `<hx>` tag like a `<h2>` to a `<h6>`. There can only be one `<h1>` tag,  in a project which should be a title for your page. 
If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.